framework/set/provisioning/providers/linode/linodeProviders.go

```diff
@@ -7,6 +7,14 @@ import (
 	"github.com/zclconf/go-cty/cty"
 )
 
+const (
```

Comment:
Why was this moved over here from `linodeMachineConfig.go`?

Reply:
azure and AWS have these consts in the providers.go file, so I just moved it to be consistent with the others.